### DISCUSSION TITLE
Problem with install the game

### DISCUSSION THREAD

**Discussion by BCJX77** [PlayCover/PlayCover#89] [Q&A] [answered]
<img width="262" alt="截屏2022-07-23 01 03 03" src="https://user-images.githubusercontent.com/99672417/180582016-e771ab64-1d45-47ec-9939-7122f17d5794.png">

**Comment by ZhichGaming** [comment, marked as answer]
You need xTools for the newest version of PlayCover. You can get it by pasting this into your terminal: 
`xcode-select --install`

**Comment by kittymai1308** [comment]
I also encounter a problem with playcover, how can I fix it?
<img width="254" alt="image" src="https://user-images.githubusercontent.com/111039550/185970969-95d784d7-d6ee-41a3-9c0f-8086f221b35e.png">

**Reply by Candygoblen123** [reply]
that is intended, playing the game that you're trying to install will result in you getting banned.

**Reply by kittymai1308** [reply]
It means that I can't play the game right?

**Reply by ZhichGaming** [reply]
That is correct.

**Reply by ZhichGaming** [reply]
Please create a separate issue next time.

**Reply by kittymai1308** [reply]
okay, thank you so much
